.github/workflows/test.yml

```diff
@@ -67,10 +67,8 @@ jobs:
     strategy:
       matrix:
         python-version: ['3.8', '3.9', '3.10']
-        django-version: ['3.2', '4.0']
+        django-version: ['4.0']
         include:
-          - python-version: '3.7'
-            django-version: '2.2'
```

Comment:
@sobolevn Here's one way to "solve" #1261, just disable testing with Django 2.2 and 3.2. 🤦

Reply:
Looks like #1329 has a better fix, reverting this.